### ISSUE TITLE
Are custom phone fields supported

### ISSUE THREAD

**Issue by JanBilton** [ringcentral/hubspot-embeddable-ringcentral-phone#221]
The app only seems to allow click to dial for numbers stored in the 'phone' property and not for custom phone properties.  For example, I have created a new property called 'Phone 2' and the RC app doesn't pick it up.

**Comment by zxdong262** [comment]
True, currently we only support native phone props: phone and mobile phone

**Comment by zxdong262** [comment]
@JanBilton I will check if it is possible to support custom phone props
